internal/worker/server.go

```diff
@@ -134,6 +138,15 @@ func notFoundHandler(writer http.ResponseWriter, request *http.Request) {
 	jsonErrorf(writer, http.StatusNotFound, "not found")
 }
 
+func (s *Server) addStatusHandler(writer http.ResponseWriter, request *http.Request, _ httprouter.Params) {
```

Comment:
`statusHandler` is probably enough, the method doesn't add anything. :-)

Reply:
👍